TaskHouseUnitTests/UsersControllerUnitTests.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Xunit;
+
+namespace TaskHouseUnitTests
+{
+    public class UsersControllerUnitTests

```

Comment:
Looks good, think you might have to run us through the details of how it works 😄